src/model/index.js

```diff
@@ -88,7 +87,12 @@ export default function(fields) {
       [fields, params] = [RETURNS_ALL_FIELDS, fields];
     }
     return new Query(model, fields, RETURNS_LIST, params);
-  }
+  };
+
+  model.meta = {
+    name: name,
+    updated: Date.now()
```

Comment:
This is going to be saved on the model class and not on the particular instance of a model unfortunately. Can  we remove updated and just save `model.name` within the class right now?